### PR TITLE
Adds Claim Card Details Component

### LINKED ISSUE #132: Create a draft of the Claim Card Details component
### Description

Component:
Step: ~Skeleton~ / Rough Draft / ~Final~ / ~Update~

### Acceptance Criteria

  - [ ] Claim Card Details component created according to the design specifications

### Context

[Component Drafting](https://docs.google.com/document/d/13amWdz6EBwFKOBbQDfXROypE8XQEWxLhUXwfA5qeEj8/edit)

Pairs well with #131 and #133

### LINKED ISSUE #133: Create a draft of the Claim Card Body component
### Description

Component:
Step: ~Skeleton~ / Rough Draft / ~Final~ / ~Update~

### Acceptance Criteria

  - [ ] Claim Card Body component created according to the design specifications

### Context

[Component Drafting](https://docs.google.com/document/d/13amWdz6EBwFKOBbQDfXROypE8XQEWxLhUXwfA5qeEj8/edit)

Pairs well with #131 and #132

### PR DESCRIPTION
**Pull request by lomky**
Adds the ClaimCard Details & Body, including placeholder values.

Claim Tracker App

===

Resolves #132 
Resolves #133 

- [ ]  Claim Card Details component created according to the design specifications
- [ ]  Claim Card Body component created according to the design specifications

### CODE REVIEW

File: stories/Main.stories.tsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import { Story, Meta } from '@storybook/react'
 import { Main } from '../components/Main'
+import { withNextRouter } from 'storybook-addon-next-router'
 
 export default {
   title: 'Component/Page Section/Main',
   component: Main,
+  decorators: [withNextRouter],
```
Comment (lomky) [inline stories/Main.stories.tsx:8]: Tangential bugfix

File: components/ClaimCard.tsx [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 export const ClaimCard: React.FC = () => {
-  const { t } = useTranslation('common')
-  const router = useRouter()
-
   return (
     <div className="claim-card">
-      <Link href="/" locale={router.locale === 'en' ? 'es' : 'en'}>
-        <Button primary label={t('change-locale')} />
-      </Link>
+      <ClaimCardDetails title="Unemployment Insurance (UI)" benefitYear="3/21/2020 - 3/20/2021" claimBalance="$0.00" />
```
Comment (kevmo) [inline components/ClaimCard.tsx:7]: Good placeholders, but do we need a ticket for an app-level state object? Not exactly about this PR, just a thought.

**Review by kevmo — APPROVED** [review]
Approved! ⛽️ 🚀 🌌

File: components/ClaimCard.tsx (hunk 2 above)
Comment (lomky) [reply components/ClaimCard.tsx:7]: dunno! how do you mean?

**Pull request merged**